src/dom/getcommonancestor.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import getAncestors from './getancestors.js';
+
+/**
+ * Searches and returns the lowest common ancestor of two given nodes.
+ *
+ * @param {Node} nodeA First node.
+ * @param {Node} nodeB Second node.
+ * @returns {Node|null} Lowest common ancestor of both nodes or `null` if nodes do not have a common ancestor.
```

Comment:
`Node` - because it returns itself if `nodeA == nodeB`. And `nodeA` can be a `Text` node.